### ISSUE TITLE
Backport fix for issue 1035

### ISSUE THREAD

**Issue by asantare** [bmuschko/gradle-docker-plugin#1066]
Latest version of Micronaut framework is using the older 6.x versions of this plugin with older docker-java client version.

See issue [1035](https://github.com/bmuschko/gradle-docker-plugin/issues/1035)

I've created a new branch starting from v6.7.0 tag with the required changes: [https://github.com/asantare/gradle-docker-plugin/tree/backport-issue-1035](https://github.com/asantare/gradle-docker-plugin/tree/backport-issue-1035)

I cannot create a PR as not hotfix branch is present in this repo

**Comment by bmuschko** [comment]
Sorry, but I am not planning to backport any changes to 6.x release. Where exactly does the Micronaut framework use the Docker plugin? Can you point me to it?

**Comment by asantare** [comment]
indeed, the best would be for Micronaut to upgrade... I didn't investigate on their side what could take to do it (I'll check that)

Here it is their usage of this library: [libs.versions.toml](https://github.com/micronaut-projects/micronaut-gradle-plugin/blob/master/gradle/libs.versions.toml)
